app/scripts/components/common/mapbox/layers/raster-timeseries.tsx

```diff
@@ -385,6 +385,7 @@ export function MapLayerRasterTimeseries(props: MapLayerRasterTimeseriesProps) {
   // FitBounds when needed
   //
   useEffect(() => {
+    if (!stacCollection || !stacCollection.length) return;
```

Comment:
```suggestion
    if (!stacCollection?.length) return;
```

Reply:
I am usually a fan of `?` but with the combination of `!` it might be hard to read?

Reply:
Good catch, thank you. 

Somehow I keep forgetting that optional chaining exists in TS. So I suggest adding <a href="https://typescript-eslint.io/rules/prefer-optional-chain/">this eslint rule</a>.

Which results in
<img width="718" alt="Screenshot 2022-10-19 at 17 17 01" src="https://user-images.githubusercontent.com/1583415/196734043-4dd61861-f90f-4530-a553-ac9e976ddb8c.png">

I suggest we do as the eslint recommends, which is using `plugin:@typescript-eslint/strict"` instead of `"plugin:@typescript-eslint/recommended"`

I tried and it would yield a very manageable number of errors (warnings actually) which I suggest I tackle on another PR

<img width="1193" alt="Screenshot 2022-10-19 at 17 22 13" src="https://user-images.githubusercontent.com/1583415/196734580-d95bf926-4a6d-4c36-8745-5922d1c05595.png">

Reply:
@nerik YES to the rule! :) Please go ahead with the PR